doc/API/API-Docs.md

```diff
@@ -1 +0,0 @@
-<meta http-equiv="refresh" content="0; url=/API" />
```

Comment:
This is to preserve old links, why remove it?

Reply:
I thought it was unused in the docs, I'll look through the original mkdocs.yml again.

Reply:
It's from the v1 theme. The links are in older installs of LibreNMS and most probably linked to from reddit and other places in old posts.

Reply:
This wasn't defined in mkdocs.yml, so it wasn't being built.  That is why I removed it.

Reply:
If you can tell me the URL it was supposed to be, I can fix it.

Reply:
It was pre-mkdocs by the looks of it:
https://github.com/librenms/librenms/issues/7210

Shows that the old links were still in place.

Reply:
You've fixed it anyway.